catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/api/WorkspaceQueryBuilder.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.monitor.api;
+
+import java.util.Set;
+import org.opengis.filter.Filter;
+
+public interface WorkspaceQueryBuilder {
```

Comment:
❓ Why did we make an interface for this? Looks like we're not registering it as a service anywhere or doing anything else with the interface  (other than creating a bean of the impl and using that)

Reply:
Mostly out of habit. But there is a test class that plugs in a mock impl.

Reply:
❗️ We should definitely remove this then.

Reply:
the interface has been removed